swagger.yaml

```diff
@@ -199,7 +199,7 @@ definitions:
         type: string
         pattern: /[A-Z][A-Z0-9_]*
         description: The name of the route in the ROUTE_NAME format
-      activatedAt:
+      activate_at:
         type: string
```

Comment:
Those are breaking changes and should therefore also change the version of the api in the swagger file